charts/ChoroplethMap.tsx

```diff
@@ -330,7 +335,6 @@ export class ChoroplethMap extends React.Component<ChoroplethMapProps> {
                 clipPath={`url(#boundsClip-${uid})`}
                 onMouseMove={this.onMouseMove}
                 onMouseLeave={this.onMouseLeave}
-                onClick={this.onClick}
```

Comment:
I think this needs to go anyways. Currently a click on a map country calls the onClick handler twice.